lte/gateway/release/build-magma.sh

```diff
@@ -407,7 +407,7 @@ $(glob_files "${ANSIBLE_FILES}/magma_modules_load" /etc/modules-load.d/magma.con
 $(glob_files "${ANSIBLE_FILES}/configure_envoy_namespace.sh" /usr/local/bin/ ) \
 $(glob_files "${ANSIBLE_FILES}/envoy.yaml" /var/opt/magma/ ) \
 $(glob_files "${ANSIBLE_FILES}/logrotate_oai.conf" /etc/logrotate.d/oai) \
-$(glob_files "${ANSIBLE_FILES}/logrotate_rsyslog.conf" /etc/logrotate.d/rsyslog.magma) \
+$(glob_files "${ANSIBLE_FILES}/logrotate_rsyslog.conf" /etc/logrotate.d/rsyslog) \
```

Comment:
I got following error after building package with this patch and deployed it.
sharing '/etc/logrotate.d/rsyslog' between multiple packages results in conflict. we shld avoid it.
the .magma file trick try to do it.

```
apt install magma=1.6.0-1622011168-f23096de
Reading package lists... Done
Building dependency tree
Reading state information... Done
The following packages were automatically installed and are no longer required:
  libnetplan0 linux-headers-5.4.0-70 linux-headers-5.4.0-70-generic linux-headers-5.4.0-71 linux-headers-5.4.0-71-generic linux-image-5.4.0-70-generic linux-image-5.4.0-71-generic linux-modules-5.4.0-70-generic
  linux-modules-5.4.0-71-generic linux-modules-extra-5.4.0-70-generic linux-modules-extra-5.4.0-71-generic
Use 'apt autoremove' to remove them.
The following packages will be upgraded:
  magma
1 upgraded, 0 newly installed, 0 to remove and 6 not upgraded.
Need to get 84.3 MB of archives.
After this operation, 24.1 MB of additional disk space will be used.
Get:1 https://artifactory.magmacore.org/artifactory/debian-test focal-1.5.0/main amd64 magma amd64 1.6.0-1622011168-f23096de [84.3 MB]
Fetched 84.3 MB in 7s (12.8 MB/s)
(Reading database ... 219323 files and directories currently installed.)
Preparing to unpack .../magma_1.6.0-1622011168-f23096de_amd64.deb ...
Unpacking magma (1.6.0-1622011168-f23096de) over (1.5.0-1621958919-c177d622) ...
dpkg: error processing archive /var/cache/apt/archives/magma_1.6.0-1622011168-f23096de_amd64.deb (--unpack):
 trying to overwrite '/etc/logrotate.d/rsyslog', which is also in package rsyslog 8.2001.0-1ubuntu1.1
Synchronizing state of redis-server.service with SysV service script with /lib/systemd/systemd-sysv-install.
Executing: /lib/systemd/systemd-sysv-install disable redis-server
Synchronizing state of dnsmasq.service with SysV service script with /lib/systemd/systemd-sysv-install.
Executing: /lib/systemd/systemd-sysv-install disable dnsmasq
Synchronizing state of lighttpd.service with SysV service script with /lib/systemd/systemd-sysv-install.
Executing: /lib/systemd/systemd-sysv-install disable lighttpd
cp: cannot stat '/etc/logrotate.d/rsyslog.magma': No such file or directory
Errors were encountered while processing:
 /var/cache/apt/archives/magma_1.6.0-1622011168-f23096de_amd64.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

Reply:
Ok, let me put that twisted logic back in. It did not feel right from the looks of it.

Reply:
we can have the .magma file in '/var/opt/magma/' to avoid confusion in log-rotate config.